client/app/projects/projects.controller.js

```diff
@@ -13,6 +14,15 @@ angular.module('observatory3App')
         }
     });
 
+    $scope.getSortOrder = function(){
```

Comment:
maybe `toggleSortOrder()`? since it is actively changing the variable and not just returning a value?